transport/src/test/java/io/scalecube/transport/TransportTest.java

```diff
@@ -38,7 +38,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 
-@SuppressWarnings("unchecked")
+@SuppressWarnings({"unchecked", "ThrowFromFinallyBlock"})
```

Comment:
Transport.stop() doesn't throw exception. I think we don't need this annotation at all.